### ISSUE TITLE
CLOUD MESSAGING: Changing google-services.json on Android won't update connections on .init

### ISSUE THREAD

**Issue by briosheje** [EddyVerbruggen/nativescript-plugin-firebase#678]
Not sure whether I've lost something, but here is what happened:

- I've created a test firebase account to test push notifications. Everything worked properly.
- Created the "production" firebase account: overwritten the .json file, rebuilt, deleted app data.

After the .init, though, the token is available on the old firebase account instead of the new one. I couldn't yet find any way to reset the configuration. Even uninstalling and wiping all the app data is not working, the token retrieved by the app always work with the old firebase profile only.

I've also checked in platforms/android/google-services.json and the file is indeed updated, though the app seems to still be linked to the old account.

Not sure whether this is a plugin-related issue or an intended firebase feature, but it's worth mentioning that if you're going to use a firebase account for production, then you should be aware that, for testing purposes, it looks like you're only able to configure it once.

**Comment by hettiger** [comment]
@briosheje Have you tried `tns platform clean android && tns platform run android` ?

**Comment by briosheje** [comment]
@hettiger I was able to find a solution.

None of `tns platform clean android && tns platform run android` worked, what I did instead is entirely wiping the android platform:

`tns platform remove android`

... And added it back again :)

`tns platform add android`

Then, of course, `tns run android`.

Now it's working properly. For anyone experiencing the issue, removing and re-adding the platform likely solves the issue!

Thanks for the support
